### PR TITLE
Intro video

### PR DESCRIPTION
**Pull request by yogeshwar607**
Intro video

### CODE REVIEW

**Review by ahhda — COMMENTED** [review]
Test purpose

File: js-exercises/abbreviate-string/abbreviateString.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+function abbreviateString(str) {
+	if (typeof str !== 'string') {
+		throw Error('argument is not of string type');
```
Comment (bharathinger) [inline js-exercises/abbreviate-string/abbreviateString.js:3]: "throw Error" should be "throw New Error"

File: js-exercises/abbreviate-string/abbreviateString.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+	}
+
+	const splitedStringArray = str.split(' ');
```
Comment (bharathinger) [inline js-exercises/abbreviate-string/abbreviateString.js:6]: 'splited' is grammatically incorrect.

**Review by bharathinger — COMMENTED** [review]
Please check the syntax of 'throw error'.

**Pull request merged**